ens-de-lyon-centre-d-ingenierie-documentaire.csl

```diff
@@ -19,7 +18,11 @@
     <category field="social_science"/>
     <category field="philosophy"/>
     <category field="literature"/>
-    <updated>2016-11-17T16:34:49+00:00</updated>
+    <category field="history"/>
+    <category field="humanities"/>
+    <category field="political_science"/>
+    <category field="sociology"/>    
```

Comment:
The CSL fields are a bit unevenly spread over the different disciplines anyway. Generally it's much easier to just search for styles by name than to filter them by discipline.